app/views/editor/rich.html.erb

```diff
@@ -518,4 +522,44 @@
       console.log("Draft:", editor.data.draft);
   }
 
+  var previewWindow = null;
+
+  function preview() {
+    let url = 'notes/preview'
+    $.ajax({
+      type: "POST",
+      url: url,
+      data:
+        {
+          'title': editor.titleModule.value(),
+          // 'image': editor.mainImageModule,
+          // 'location': editor.titleModule.value(),
```

Comment:
Similary location tags are added when we click `publish`, however we can read the location from map module

Reply:
tags can be passed as `tags: lat:1.324592,  lon:13.687162,  zoom:5`, and on preview the map will be rendered automatically.

Reply:
I have managed to figure out how to access the location...

this is the current POST data that I'm sending
```suggestion
               data:
        {
          'title': editor.titleModule.value(),
          'body': editor.richTextModule.value(),
          'main_image': image_id,
          'tags': editor.tagsModule.value(),
          'latitude': editor.mapModule.blurredLocation.getLat(),
          'longitude': editor.mapModule.blurredLocation.getLon(),
          'zoom': editor.mapModule.blurredLocation.getZoom()
        },
```